rpc/connection.go

```diff
@@ -336,7 +336,12 @@ type ConnectionOpts struct {
 	// InitialReconnectBackoffWindow, if non zero, causes a random backoff
 	// before reconnecting. The random backoff timer is fast-forward-able by
 	// passing in a WithFireNow(ctx) into a RPC call.
-	InitialReconnectBackoffWindow time.Duration
+	InitialReconnectBackoffWindow func() time.Duration
+	// As the name suggests, we normally skip the "initial REconnect backoff"
```

Comment:
s/REconnect/reconnect

Reply:
I have a slight preference for an interface here which includes a `GetReconnectBackoffWindow()` method. But I'm OK with this too.

Please update the comment to reflect that this field is now a function.

The other thing is that I actually don't remember why this was named `InitialReconnectBackOffWindow` rather than `ReconnectBackOffWindow`. Maybe I planned to have it changeable, but that doesn't seem to be the case anymore, so the naming is a bit confusing now. So if you think the latter is better, feel free to change it!

Reply:
The reason it's not `ReconnectBackOffWindow` is because there's also a `ReconnectBackoff` function immediately above. Perhaps the Right Thing would be to combine the two, but I tried not to change more than I needed to here.